<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjCenterDialog</class>
 <widget class="QDialog" name="ProjCenterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>70</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>40</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>70</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>设置投影中心</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>经度：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit_Lon"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>纬度：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="lineEdit_Lat"/>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="pushButton_OK">
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
